l10n/configs/pontoon.toml

```diff
@@ -327,3 +327,11 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/platform-post-download.ftl"
     l10n = "{locale}/products/vpn/platform-post-download.ftl"
+[[paths]]
+    reference = "en/mozorg/try-picture-in-picture.ftl.ftl"
+    l10n = "{locale}/mozorg/try-picture-in-picture.ftl.ftl"
+    locales = [
```

Comment:
DE and FR are handled by the vendor now so I think Pontoon should only get IT. 

Right now vendor.toml is set up to send ALL strings to the entire (short) list of locales but if we want to limit it to only DE and FR I think we could add this file with its own rule. But then we can also just let it go to all the vendor locales without doing anything special.